src/applications/disability-benefits/all-claims/sass/disability-benefits.scss

```diff
@@ -12,3 +12,11 @@
 .full-page-alert {
   margin-bottom: 2em;
 }
+
+button {
+  margin-top: 1em;
+}
+
+.list-four {
```

Comment:
Alternatively, `.process-step:last-child` should do the trick as well (and scale better, though that's not strictly speaking necessary here).

Reply:
Oh hey, that's a thing!